ci/ci.py

```diff
@@ -388,45 +382,49 @@ def main(argv):
     branch = get_branch()
 
     # configure build flags based on the branch
+    archive_channel = None
+    editor_update_channel = None
     release_channel = None
     skip_editor_tests = False
     if branch == "master":
-        engine_channel = "stable"
-        editor_channel = "editor-alpha"
-        release_channel = "editor-stable"
+        archive_channel = "stable"
+        editor_update_channel = "editor-alpha"
```

Comment:
Tbh, I wonder about this.
Between the beta period start and the release, the editor-alpha channel might have been updated with other editor fixes. If we now push an _older_ editor to that channel, we'll revert the changes for our users, and they'll have to wait for another push to the editor-dev branch.

Am I wrong?